projects.html

```diff
@@ -0,0 +1,60 @@
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@500&family=Sometype+Mono:wght@700&display=swap" rel="stylesheet">
+    <link rel="icon" href="_assets/logo.png">
+</head>
+<body style="margin: 0; display: flex; flex-direction: column; height: 100vh;">
+    <!-- Top bar section -->
+    <div class="topTile">
+        <div class="topbar">
+            <div class="topbarLeft">
+                <img src="_assets/logo.png" width="100" height="100" alt="AutoSlug" style="padding-left: 10px">
+            </div>
+            <div class="topbarRight">
+                <a href="index.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">HOME</a>
+                <a href="about.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">ABOUT</a>
+                <a href="projects.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">PROJECTS</a>
+                <a href="team.html" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none">TEAM</a>
+                <a href="https://secure.ucsc.edu/s/1069/bp18/interior.aspx?sid=1069&gid=1001&pgid=780&cid=1749&dids=1207" class="links" style="padding-left: 15px; padding-right: 15px; text-decoration:none" target="_blank">DONATE</a>
+            </div>
+        </div>
+    </div>
+    <!-- Page Title -->
+    <div class="title" style="background-color:rgb(72, 9, 116);padding-top: 20px;">OUR PROJECTS</div>
+    <div class="centerTile" style="background-color:rgb(72, 9, 116); flex: 1 0 auto; display: flex; flex-direction: column; justify-content: center;">
+        <!-- Project 1 -->
+        <div class="midbox" style="padding: 40px; display: flex;">
+            <img src="_assets/IMG_3494.PNG" width="700px" height="400px" style="padding-right: 30px;">
+            <div style="font-family: 'Orbitron', sans-serif; font-size:20px; color:white; padding-top: 40px; padding-left: 20px; padding-right: 20px;">
+                <h2>ASL</h2>
+                <p>Our team is developing a refined computer vision model to accurately interpret the intricate hand and body movements of American Sign Language (ASL). This advanced technology meticulously analyzes the subtle nuances of ASL signs, enabling real-time translation into standard English text. By bridging the communication gap between the deaf and hearing communities, our model enhances inclusivity and facilitates seamless interactions across linguistic barriers.</p>
+            </div>
+        </div>
+        <!-- Project 2 - Flipped -->
+        <div class="midbox" style="padding: 40px; display: flex; flex-direction: row-reverse;">
+            <img src="_assets/modbot.webp" width="600px" height="400px" style="padding-left: 30px;">
```

Comment:
the photo got squished horizontally bc it's not 600x400, which you can fix by removing one of those dimensions (html will automatically resize the other dimension). you should do that for all photos (make sure you aren't specifying both the width and height, and instead you should just specify the max-width)

Reply:
also, you should use pngs instead of webp pls :)

Reply:
oh and imo you should use percentage values instead of hard set pixel values for the width of images, because it makes it more likely to look similar on everyone's screens and not have weird aspect ratios on certain screen sizes (that being said, you should set min-width in pixels so it doesn't shrink beyond some amount, and I think there's css tags you can use to say "if the screen is less than X px wide, make the image 100% wide"

Reply:
This is also still there